packages/mosaic/tree/tree-option.component.ts

```diff
@@ -82,6 +82,10 @@ export class McTreeOption extends CdkTreeNode<McTreeOption> implements OnInit, O
 
     private _disabled: boolean = false;
 
+    get showCheckbox(): boolean {
+        return this.multiple && this.tree.multipleMode === MultipleMode.CHECKBOX;
```

Comment:
are you sure that this need check `this.multiple` ?

Reply:
fixed